external/fv3fit/tests/test_sklearn_wrapper.py

```diff
@@ -211,8 +204,24 @@ def test_predict_columnwise_is_deterministic(regtest):
     dims = ["x", "y", "z"]
     shape = (2, 2, nz)
     arr = np.arange(np.prod(shape)).reshape(shape)
-    data = xr.Dataset({"a": (dims, arr), "b": (dims, arr + 1)})
-    wrapper.fit([data])
+    input_data = xr.Dataset({"a": (dims, arr), "b": (dims, arr + 1)})
+    wrapper.fit([input_data])
+    return input_data, wrapper
+
 
-    output = wrapper.predict_columnwise(data, feature_dim="z")
+def test_predict_is_deterministic(regtest):
+    """Tests that fitting/predicting with a model is deterministic
+
+    If this fails, look for non-deterministic logic (e.g. converting sets to lists)
+    """
+    input_data, wrapper = fit_wrapper_with_columnar_data()
+    output = wrapper.predict(input_data)
     print(joblib.hash(np.asarray(output["b"])), file=regtest)
+
+
+def test_predict_returns_unstacked_dims():
+    # 2D output dims same as input dims
+    input_data, wrapper = fit_wrapper_with_columnar_data()
```

Comment:
In case fit_wrapper_with_columnar_data gets updated?
```suggestion
    input_data, wrapper = fit_wrapper_with_columnar_data()
    assert len(input_data.dims) > 2
```